resources/views/people/edit.blade.php

```diff
@@ -44,12 +44,21 @@
                   v-bind:title="'{{ trans('people.people_add_lastname') }}'">
                 </form-input>
               </div>
+              <div class="dtc-ns pb0-ns w-100">
+                <form-input
+                  value="{{ $contact->nickname }}"
+                  v-bind:input-type="'text'"
+                  v-bind:id="'nickname'"
+                  v-bind:required="false"
+                  v-bind:title="'{{ trans('people.people_add_nickname') }}'">
+                </form-input>
+              </div>
             </div>
 
             @else
```

Comment:
What about the other name_order display ?

Reply:
@asbiin I take into account two types: `firstname + lastname` and `lastname + firstname` in the edit screen. There is also always the `nickname` field because reasons.